era/front.js

```diff
@@ -131,3 +150,508 @@ function setParamAndReload() {
 function redirectToThreat() {
   location.href = location.origin + location.pathname + "/threat";
 }
+/**
+ * @param {string} e
+ */
+function printError(e) {
+  console.log(e);
+  alert("Error:\n" + e + "\n\nRefresh the page to start again.");
+}
+
+/**
+ * @type {Record<string, Report>}
+ */
+const reports = {};
+
+/**
+ * @param {import("./base.js").GameVersionConfig} config
+ */
+export function selectReport(config) {
```

Comment:
All the `selectX()` functions now take `config`, so that we can supply the config from the game version without using globals.

I could undo this decision, but I think its better to see clearly which things need config and where it comes from